LiskHQ/lisk-mobile
# Implement the wallet page for other accounts - Closes #435

# What was the bug or feature?
Described in #435 

## Type of change
- Enhancement (a non-breaking change which adds functionality)

### How to test it?
 - Infinite scroll should work, adding older tx to the end of the list.
 - Pull to refresh should add the new tx to the beginning of the list.
 - Bookmark and send to buttons should work.

# Checklist:
- My code follows the style guidelines of this project
- I have performed a self-review of my own code
- I have commented my code, particularly in hard-to-understand areas
- I have made corresponding changes to the documentation
- I have added tests that prove my fix is effective or that my feature works
- New and existing unit tests pass locally with my changes
